src/player-data.ts

```diff
@@ -150,7 +154,7 @@ export default class PlayerData extends EventEmitter {
   }
 
   spendablePowerTokens(): number {
-    return this.power.area3 + this.brainstoneValue();
+    return this.power.area3 * this.tokenModifier + this.brainstoneValue();
```

Comment:
I would do `return Math.floor(this.power.area3 * this.tokenModifier) + this.brainstoneValue();`